EF.Extensions.PgCopy/DbContextExtensions.cs

```diff
@@ -159,7 +158,9 @@ private static object PostgreSqlCopyHelperFactory(IEntityType entityType)
 
             var copyHelper = c.Invoke(new object[] {entityType.GetSchema(), QuoteIdentifier(entityType.GetTableName())});
 
-            var mapMethodInfo = copyHelper.GetType().GetMethod("Map", BindingFlags.Instance | BindingFlags.Public);
+            var mapMethodInfo = copyHelper.GetType().GetMethods().First(n =>

```

Comment:
This is horrible and it should be possible to use a GetMethod overload that takes an array of types. I couldn't get it to play ball though.